internal/handlers/pg/pgdb/indexes.go

```diff
@@ -204,8 +204,15 @@ func createPgIndexIfNotExists(ctx context.Context, tx pgx.Tx, schema, table, ind
 			return lazyerrors.Errorf("unknown sort order: %d", field.Order)
 		}
 
-		// It's important to sanitize field.Field data here, as it's a user-provided value.
-		fieldsDef[i] = fmt.Sprintf(`((_jsonb->%s)) %s`, quoteString(field.Field), order)
+		// if the key is foo.bar, then need to modify it to foo -> bar
+		fs := strings.Split(field.Field, ".")
+		transformedParts := make([]string, len(fs))
+
+		for i, f := range fs {
```

Comment:
Let's just `j` here, `i` is used in the outer loop and it's easier to follow if they don't share the same loop variable name :pray: